lib/Pipeline.js

```diff
@@ -105,8 +107,17 @@ Pipeline.processJSONWithExtras  = function(gltfWithExtras, options) {
         bakeAmbientOcclusion(gltfWithExtras, aoOptions);
     }
 
-    // Run removeAll stage again after all pipeline stages have been run to remove objects that become unused
+    // Run through light optimization for newly created data
+    MergeDuplicateProperties.mergeAccessors(gltfWithExtras);
+    removeDuplicatePrimitives(gltfWithExtras);
+    combineNodes(gltfWithExtras);
+    combineMeshes(gltfWithExtras);
+    MergeDuplicateProperties.mergeAccessors(gltfWithExtras);
+    removeDuplicatePrimitives(gltfWithExtras);
+    combinePrimitives(gltfWithExtras);
+    combineMeshes(gltfWithExtras);
```

Comment:
I think we talked offline about this a while ago, but I forget the conversation...

If possible we should try to avoid the second cleanup, and ideally our stages shouldn't be producing unoptimized gltf's. Sorry if I forgot the reason behind this.